### ISSUE TITLE
Websockets improvements

### ISSUE THREAD

**Issue by iangregsondev** [firecamp-io/Firecamp#91]
Great tool!

I wonder if could make a suggestion, the listeners, maybe they should be an option to LISTEN automatically on connection and / or a button to say "Listen all" 

Also ordering alphabetically the names of the listeners.

Also once connected I don't see a button to disconnect and / or reconnect - this would be really cool also.

It also would be really great if its possible to send down (sequentially) saved emitters.

For example, when I connect I have call 2 emitters, its mandatory. i.e.

![image](https://user-images.githubusercontent.com/2370364/76705867-a3352580-66e3-11ea-9c58-d8f35ed89e5f.png)

Would be really cool if on a connect / reconnection - then it could call these 2 emitters that i saved.

also would be nice to have these events for listening / unlistening into its own panel 

![image](https://user-images.githubusercontent.com/2370364/76705891-dc6d9580-66e3-11ea-87f0-ab33c61743a5.png)

And finally, having all the websocket messages output to the main center works but after quite a few messages coming through - you kind of lose them.

It would also be nice to have an option to put them into another tab or something similar so you can separate and identify the events with the listeners and also filter on them

![image](https://user-images.githubusercontent.com/2370364/76705922-13dc4200-66e4-11ea-87c6-ab44501d1c31.png)

These are my ideas above and i am certainly not saying I am right but its what i found while working with it.

Great product - keep up the great work.

Thanks!

**Comment by Nishchit14** [comment]
It'll definitely help us to design v2 for WS/SocketIO. Thanks for your suggestions @iangregsondev

**Comment by evandroabukamel** [comment]
Hello, everyone. I have a suggestion too. On the argument's value it would be good to have other types of parameters, like number and bool. 

![image](https://user-images.githubusercontent.com/6452880/82161496-9a8fd580-9873-11ea-8634-760f015d90ef.png)

I'm testing a websocket and one on the parameters is expect to be a number but Firecamp sends a string, so it doesn't work. This websocket is created by an external JS package and I needed to change it's code to convert the string to number to make everything work. I opened an issue there, and I'm doing the same here. I believe that problem is not going to happen when I'm implement my client as I can send typed parameters, but in Firecamp I'm not able to send a number.

Thanks.
